File: Java/Cristian Maldonado/Bitwise/Opposite_Signs.java
public class Opposite_Signs {

    static boolean opposite(int a, int b){
        return (a ^ b) < 0;
    }
    
    public static void main(String[] args) {
       int a = 123;
       int b = -123;
       if(opposite(a, b)){
           System.out.println("Distintos signos");
       }else{
           System.out.println("Iguales signos");
       }
    }   
}

File: Codigos estudiantes por lenguaje/C++/Cristian Maldonado/Bitwise/Opposite_Signs.cpp
#include <iostream>

using namespace std;

bool opposite(int a, int b){
    return (a ^ b) < 0;
}

int main()
{
    int a = 123;
    int b = -123;
    if(opposite(a, b)){
        cout<<"Distintos signos"<<endl;
    }else{
        cout<<"Iguales signos"<<endl;
    }
    return 0;
}
